spec/presenters/content_item_presenter_spec.rb

```diff
@@ -39,5 +39,13 @@
 
       expect(attributes["links"]["organisations"]).to eq([content_id])
     end
+
+    it "includes linked people with a policy area" do
+      content_id = SecureRandom.uuid
+      policy_area = FactoryGirl.create(:policy_area, people_content_ids: [content_id])
+      attributes = ContentItemPresenter.new(policy_area).exportable_attributes.as_json
```

Comment:
Could you switch this to `to_json`?  Part of my work is changing some of these specs to not require Rails, and `as_json` is a Rails addon.